docs/modules/ROOT/pages/how-tos/ingress/self-signed-ingress-cert.adoc

```diff
@@ -0,0 +1,99 @@
+= Self-signed default Ingress certificate
+
+[NOTE]
+--
+Steps to implement a self-signed default Ingress certificate for the OpenShift Router. This isn't meant to be used in production!
+
+These steps follow the https://docs.openshift.com/container-platform/4.6/security/certificates/replacing-default-ingress-certificate.html#replacing-default-ingress_replacing-default-ingress[Replacing the default ingress certificate] docs to set up a regular commercial certificate.
+--
+
+== Generate a self-signed ingress certificate
+
+A private key and certificate is generated using the https://www.openssl.org[openssl] command line tool. OpenShift requires the configuration of the Subject Alternative Name (SAN). The distinguished name (DN) Common Name (CN) must be equal to the SAN wildcard domain, in example `*.apps.<cluster>.<domain>.<tld>`.
+
+. Create the private key `ingress.key` and the certificate `ingress.crt` in a single step:
++
+[source,console]
+----
+openssl req -x509 -out ingress.crt -keyout ingress.key -config <(
+cat <<-EOF
+[ req ]
+prompt = no
+encrypt_key = no
+default_bits = 4096
+default_md = sha256
+distinguished_name = dn
+x509_extensions = v3_req
+
+[ dn ]
+C = AU
+ST = Some-State
+O = Internet Widgits Pty Ltd
+CN = *.apps.<cluster>.<domain>.<tld>
+
+[ v3_req ]
+keyUsage = digitalSignature, keyEncipherment
+extendedKeyUsage = serverAuth
+basicConstraints = critical,CA:false
+subjectKeyIdentifier = hash
+subjectAltName = @alt_names
+
+[ alt_names ]
+DNS.1 = *.apps.<cluster>.<domain>.<tld>
+EOF
+)
+----
+
+. Verify the wildcard certificate has been created in a valid form with a SAN:
++
+[source,console]
+----
+openssl x509 -noout -text -in ingress.crt | grep -A1 'X509v3 Subject Alternative Name'
+----
+
+== Configure the generated self signed Ingress certificate in OpenShift
+
+[NOTE]
+--
+Because the self-signed certificate has been created without a CA, the certificate itself is stored in the `custom-ca` ConfigMap. This ensures the created certificate is accepted as valid.
```

Comment:
```suggestion
Because the self-signed certificate has been created without a CA, the certificate itself is stored in the `custom-ca` ConfigMap.
This ensures the created certificate is accepted as valid.
```